### PR TITLE
add suport to extra volumes and also bump to the latest mm version

### PR DESCRIPTION
**Pull request by cpanato**
- Adding support to user can add extra volumes
- bump the mm version

### CODE REVIEW

**Comment by cpanato** [comment]
it is good to have tests :)

File: charts/mattermost-enterprise-edition/values.yaml [hunk 1]
```diff
@@ -156,7 +157,20 @@ mattermostApp:
...
+  extraVolumes: []
+    # - hostPath:
+    #     path: /var/log
+    #   name: varlog
+  extraVolumeMounts: []
+    # - name: varlog
+    #   mountPath: /host/var/log
+    #   readOnly: true
+
+# Mysql HA Section. Use this to configure the mysql.
```
Comment (jwilander) [inline charts/mattermost-enterprise-edition/values.yaml]:
```suggestion
# Mysql HA Section. Use this to configure mysql.
```

File: charts/mattermost-enterprise-edition/values.yaml [hunk 2]
```diff
@@ -201,6 +215,8 @@ mattermost-elasticsearch:
   data:
     replicaCount: 2
 
+# Minio Section. Use this to configure the minio storage.
+# If you want an external storage please disable this
```
Comment (jwilander) [inline charts/mattermost-enterprise-edition/values.yaml]:
```suggestion
# If you want to use external storage please disable this
```

File: charts/mattermost-enterprise-edition/values.yaml [hunk 3]
```diff
@@ -156,7 +157,20 @@ mattermostApp:
     #     mountPath: /mattermost/plugins/
     #   - name: mattermost-plugins-client
     #     mountPath: /mattermost/client/plugins/
+    #   Add here the extra volumes you might want to use. Use the values you define in the extraVolumeMounts
```
Comment (jwilander) [inline charts/mattermost-enterprise-edition/values.yaml]:
```suggestion
    #   Add the extra volumes you might want to use here. Use the values you define in the extraVolumeMounts
```

File: charts/mattermost-enterprise-edition/values.yaml [hunk 4]
```diff
@@ -86,11 +86,12 @@ global:
         # More details at https://docs.mattermost.com/mobile/mobile-hpns.html.
         useHPNS: false
 
+# Mattermost deployment section. Use to configure the deployment and how many replicas and so on.
```
Comment (jwilander) [inline charts/mattermost-enterprise-edition/values.yaml]:
```suggestion
# Mattermost deployment section. Use to configure the deployment, how many replicas and other fields.
```

File: charts/mattermost-enterprise-edition/values.yaml [hunk 5]
```diff
@@ -156,7 +157,20 @@ mattermostApp:
     #     mountPath: /mattermost/plugins/
     #   - name: mattermost-plugins-client
     #     mountPath: /mattermost/client/plugins/
+    #   Add here the extra volumes you might want to use. Use the values you define in the extraVolumeMounts
 
+  # Add additional volumes and mounts, for example to add saml keys in the app or any other thing
```
Comment (jwilander) [inline charts/mattermost-enterprise-edition/values.yaml]:
```suggestion
  # Add additional volumes and mounts, for example to add SAML keys in the app or other files the app server may need to access
```

**Comment by cpanato** [comment]
@jwilander done

**Comment by jwilander** [comment]
@cpanato this is good to merge right?

**Pull request merged**